charts/incubator/plex-meta-manager/templates/_secret.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* Define the secret */}}
+{{- define "pmm.secret" -}}
+
+{{- $secretName := printf "%s-secret" (include "tc.common.names.fullname" .) }}
+{{- $pmm := .Values.pmm -}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $secretName }}
+  labels:
+    {{- include "tc.common.labels" . | nindent 4 }}
+stringData:
+  {{/* PMM */}}
+  PMM_CONFIG: "/config/config.yml"
+  PMM_TIME: {{ join "," $pmm.times | quote }}
+  {{- with $pmm.run }}
+  PMM_RUN: {{ . | quote }}
+  {{- end }}
+  {{- with $pmm.no_countdown }}
+  PMM_NO_COUNTDOWN: {{ . | quote }}
+  {{- end }}
```

Comment:
```suggestion
  {{- if not $pmm.run }}
  PMM_TIME: {{ join "," $pmm.times | quote }}
  PMM_NO_COUNTDOWN: {{ $pmm.no_countdown | quote }}
  {{- end }}
  PMM_RUN: {{ $pmm.run | quote }}  
```

with BOOLEAN or if BOOLEAN
it will only enter if it's true.
When false it will never assign the false value to the Env variable.